Library/Formula/artifactory-cli-go.rb

```diff
@@ -0,0 +1,23 @@
+class ArtifactoryCliGo < Formula
+  desc "Artifactory-CLI provides a command-line interface."
```

Comment:
Please remove `Artifactory-CLI` from the beginning of the description: `desc "command-line interface to ...`